### ISSUE TITLE
Issues with flex grid system/tailwind CSS grid system

### ISSUE THREAD

**Issue by Zer0xxxx** [nunocoracao/blowfish#161]
Hello,

I experience some issues when I try to use some Tailwind CSS properties to build grids on the homepage.
All the elements are displayed the one above the others. :/

I don't see any example including grids (except with _markdown tables_, but I don't find how to _v-center_ elements with this way) in the example site, nor in websites built with _Blowfish_.

**EDIT: I found out how to deal with it. I still think it's somehow ugly and too tricky and it would be awesome to be standardized in the theme to stay consistent.**

Regards.

**Comment by nunocoracao** [comment]
Hello @Zer0xxxx Blowfish and Hugo allow users to override most of the default HTML, add custom CSS, and custom javascript code. Solving this issue would require massive changes to the core code of the theme which I don't think brings much value at this stage.

**Comment by Zer0xxxx** [comment]
Hello @nunocoracao 

I misspoke.
Please, let me rephrase my idea: I think it would be a good idea to include examples on the _Blowfish documentation_ about how to make successfully grids on a Hugo website using _Blowfish_, with the _Blowfish_ standards.

That's all.

**Comment by Zer0xxxx** [comment]
Sorry @nunocoracao, I can't write English properly today for some unknown reasons.
I have edited my previous message.

**Comment by nunocoracao** [comment]
Hello @Zer0xxxx, thanks for your commentary. Blowfish is not meant to be a CSS framework, the goal here is not for people to override CSS and start creating an entirely new grid. However, Hugo already supports those use-case, in the sense that, you can override all my HTML code and turn it into whatever you want. So, in summary, this ticket will remain closed because:
1- functionality requested is already available via Hugo by overriding theme overlays
2- Blowfish is meant to be a theme for Hugo, not a full-baked CSS framework

**Comment by Zer0xxxx** [comment]
Okay, no problem!
